config/routes.rb

```diff
@@ -1,18 +1,26 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: {registrations: 'registrations'}
   root to: 'companies#index'
+  get 'company_favourites/create'
+  get 'developer_favourites/create'
```

Comment:
Been a while since I bother with routes, but aren't these the samething as calling 
`resources :company_favroutes, only: [ :create ]` 
as below?